soc/renesas/ra/ra8/Kconfig.soc

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2024 Renesas Electronics Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+
+config SOC_SERIES_RA8
+	bool
+	select SOC_FAMILY_RENESAS_RA
+	help
+		Renesas RA8 series
+
```

Comment:
Even if the Renesas RA series belongs to the same series in the catalog, the configuration may vary greatly. 
For example, the peripheral register configuration of the RA4M1 is very different from that of the RA4M2 and is rather similar to the RA6M1.
(I have no idea why it's like this...)

For this reason, it seems better to divide the series here into groups on the catalog (RA8M1, RA6E1, ...) rather than series on the catalog (RA8, RA6, ...).

Reply:
@soburi : 

Our first intention when group series on the catalog (RA8, RA6, ...) is to avoid some duplicated codes (as you can see, in RA8 series, there are a few differences between RA8M1, RA8D1 and RA8T1) and follow Renesas RA Family MCU Portfolio (described at: https://www.renesas.com/us/en/products/microcontrollers-microprocessors/ra-cortex-m-mcus).

We also agree with you that it will be difficult to deduce unify rules for this "group". It will make codes become complicated. So, after discussion, we made decision to update as your proposal. Please check the update and thank you so much.